tests/ios/pages/ios_register.py

```diff
@@ -349,10 +349,14 @@ def select_country(self):
             webdriver elements: country search field
         """
 
-        country_field = self.global_contents.wait_and_get_element(
+        # country_field = self.global_contents.get_element_by_label_ios(
+        #     self.driver,
```

Comment:
do we need these comments?
If yes, then add context here.

Reply:
it was test code and left during code cleaning. thank you for pointing it out. 
Fixed now.